tests/spark/test_mrjob_spark_harness.py

```diff
@@ -13,9 +13,20 @@
 # limitations under the License.
 """Test the Spark Harness."""
 import uuid
+import os
+import random
+import string
+import sys
+import json
 from io import BytesIO
 from os.path import join
+from collections import defaultdict
+from contextlib import contextmanager
+from io import StringIO
+from tempfile import gettempdir
+from shutil import rmtree
 
+from mrjob.examples.mr_word_freq_count import MRWordFreqCount
```

Comment:
There probably shouldn't be two of this line (but I always clean stuff like this out at release time anyhow).